src/bitmap/inherent.rs

```diff
@@ -43,6 +43,102 @@ impl RoaringBitmap {
         container.insert(index)
     }
 
+    /// Inserts a range of values from the set specific as [start..end). Returns
+    /// the number of inserted values.
+    ///
+    /// Note that due to the exclusive end this functions take indexes as u64
+    /// but you still can't index past 2**32 (u32::MAX + 1).
+    ///
+    /// # Safety
+    ///
+    /// This function panics if the range upper bound exceeds `u32::MAX`.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use roaring::RoaringBitmap;
+    ///
+    /// let mut rb = RoaringBitmap::new();
+    /// rb.insert_range(2..4);
+    /// assert!(rb.contains(2));
+    /// assert!(rb.contains(3));
+    /// assert!(!rb.contains(4));
+    /// ```
+    pub fn insert_range(&mut self, range: Range<u64>) -> u64 {
```

Comment:
Was wondering about this API, maybe one day we will be able to use the `RangeBounds` trait, the only blocking point is: Do we want to accept unbounded ranges? I am not against in fact and with the `insert_range` and `delete_range` API, it can be efficient!

Reply:
Yeah I was thinking (and will keep thinking!) about `RangeBounds` - it definitely feels wrong having `Range<u64>`!